src/corehost/cli/deps_resolver.cpp

```diff
@@ -89,8 +63,41 @@ void add_unique_path(
     existing->insert(real);
 }
 
+pal::string_t get_deps_filename(const pal::string_t& path)
+{
+    if (path.empty())
+    {
+        return path;
+    }
+
+    auto name_pos = path.find_last_of('/');
+    if (name_pos == pal::string_t::npos)
+    {
+        return path;
+    }
+
+    return path.substr(name_pos + 1);
+}
+
 } // end of anonymous namespace
 
+  // -----------------------------------------------------------------------------
+  // A uniqifying append helper that doesn't let two entries with the same
+  // "asset_name" be part of the "output" paths.
```

Comment:
What is the `output` paths here?  The parameter that was `output` removed.

Reply:
Update comment. Thanks